src/middlewares.js

```diff
@@ -360,8 +361,15 @@ export function handleParseErrors(err, req, res, next) {
       default:
         httpStatus = 400;
     }
+    // capitalize the message if the error code isn't missing field field
+    if (err.code !== 142) {
```

Comment:
I'm not sure we should introduce a change in capitalization for a single error message. Not only may this be a breaking change for some deployments, but it is also missing a broader concept of how error messages should be styled across the server.